src/SharedWeb/Utilities/ServiceCollectionExtensions.cs

```diff
@@ -727,4 +727,15 @@ public static void AddWebAuthn(this IServiceCollection services, GlobalSettings
             });
         });
     }
+
+    public static void AddOptionality(this IServiceCollection services,
```

Comment:
Returning the `IServiceCollection` is generally standard so calls can be chained. We don't consistently follow it but I prefer it.